version_control/katalyst_by_codurance/css/modules/benefits-listing.css

```
@@ -5,47 +5,68 @@
     margin-inline: auto;
 }
 
-.benefits__listing li {
-    text-align: center;
+.benefit {
+    display: flex;
+    gap: var(--space-m);
 }
 
-.benefits__listing i {
-    color: var(--orange);
-    font-size: var(--fs-5);
+
+.benefit__icon {
+    max-width: 100px;
+    flex-basis: 15%;
+    position: relative;
+}
+
+
+.benefit__icon i {
+    font-size: var(--fs-4);
+    transform: translateX(-50%);
+    left: 50%;
+    position: absolute;
+}
+
+.icon-background {
+    position: absolute;
+    top: 0;
+    z-index: -1;
 }
 
-.benefits__listing h3,
-.benefits__listing p {
-    color: white;
+.benefit__content {
+    flex-basis: 125%;
 }
 
-.benefits__listing h3 {
+.benefit__content h3 {
     font-size: var(--fs-1);
     margin-top: var(--space-2xs);
     font-family: {{ theme.text.body_font.font.font }};
     font-weight: bold;
 }
 
+.benefit__content p {
+    font-weight: var(--fw-medium);
+}
+
 {% call utils.medium_large_and_extra_large() %}
     .benefits__listing {
-        display: flex;
-        gap: var(--space-2xl);
-        justify-content: center;
+        display: grid;
+        grid-template-columns: 1fr 1fr;
+        gap: var(--space-xl);
```

The benefit items on the bigger viewpoints seem to be somewhat larger and closer than on the design; I'm not sure if it would be the perfect solution, but maybe making this gap bigger helps. I've tried with --space-3xl and seems to improve it a bit.

I'm happy to have another solution if you find a better way though!
```suggestion
        gap: var(--space-3xl);
```